docs/sources/_index.md.t

```diff
@@ -54,30 +58,18 @@ hero:
 
 Collecting the relevant telemetry data, such as metrics, logs, and traces, for analysis is an indispensable part of understanding the health of your system.
 
-{{< param "PRODUCT_NAME" >}} is a multifunction tool you can use to collect, process, and forward telemetry data to the observability backend of your choosing.
+{{< param "PRODUCT_NAME" >}} is more than just a collector. You can use {{< param "PRODUCT_NAME" >}} to collect, process, and forward telemetry data to the observability backend of your choosing.
 
 {{< param "PRODUCT_NAME" >}} has the following features to help you customize, scale, secure, and troubleshoot your data pipeline.
-1. Custom components
-1. GitOps compatibility
-1. Clustering support
-1. Security
-1. Debugging utilities
 
-Check out the {{< param "PRODUCT_NAME" >}} [Introduction] page for more information on these and other key features.
+* Custom components
+* GitOps compatibility
+* Clustering support
+* Security
+* Debugging utilities
 
 {{< param "PRODUCT_NAME" >}} is flexible, and you can easily configure it to fit your needs in on-prem, cloud-only, or a mix of both.
 
```

Comment:
One thing to add here is a clear call to action, which is for them to install, configure, and collect forward telemetry data.

We could do that by including the section on getting started in the original draft on the overview page

OR

include a blurb saying, "Getting started with Alloy consists of 3 major steps.
Click here to get started." Then, link it to a new page with content in the original draft as shown below.

New page:
Getting started with Alloy consists of 3 major steps:

Install {{< param "PRODUCT_NAME" >}}
Configure {{< param "PRODUCT_NAME" >}}
Collect and forward telemetry data to the observability backend of your choice.
In addition, you can use the Grafana dashboard to visualize the data collected from the app or infrastructure.

For a quick overview of this process, check out the following tutorials.

[Use Grafana Alloy to send logs to Loki](https://grafana.com/docs/alloy/latest/tutorials/send-logs-to-loki/)
[Use Grafana Alloy to send metrics to Prometheus](https://grafana.com/docs/alloy/latest/tutorials/send-metrics-to-prometheus/)
For more tutorials, check out the [tutorial page](https://grafana.com/docs/alloy/latest/tutorials/)
What do you think?

Reply:
It's normally clear to users that they have to install and configure it. The main issue is that there are many ways to do so. For example, do they have to deploy one Alloy per server? Or one central Alloy instance? What if they use OpenTelemetry? Or Prometheus? I think we should simply link to several articles which help them get started quickly with the sort of telemetry they are interested in. The ones you linked to are a good choice:

[Use Grafana Alloy to send logs to Loki](https://grafana.com/docs/alloy/latest/tutorials/send-logs-to-loki/)
[Use Grafana Alloy to send metrics to Prometheus](https://grafana.com/docs/alloy/latest/tutorials/send-metrics-to-prometheus/)[Use Grafana Alloy to send logs to Loki](https://grafana.com/docs/alloy/latest/tutorials/send-logs-to-loki/)
[Use Grafana Alloy to send metrics to Prometheus](https://grafana.com/docs/alloy/latest/tutorials/send-metrics-to-prometheus/)

I'd also add one more about collecting OpenTelemetry. I do also wonder if we should rename those pages like this
* send metrics to Prometheus -> collect infrastructure metrics
* send logs to Loki -> collect infrastructure logs
* send OpenTelemetry signals -> collect application telemetry

This reflects the [current guidance](https://grafana.com/docs/alloy/latest/collect/choose-component/) on how to use components.

Reply:
@ptodev, this is such a great point. 

@clayton-cornell  recommended that we do not add a CTA to an overview page, which makes sense.

I suggest we include this information on the get started page: https://grafana.com/docs/alloy/latest/get-started/

Why don't we touch base on this when we restructure and improve the getting started page of the doc?